drivers/clock_control/clock_control_mspm0.c

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2025 Texas Instruments
+ * Copyright (c) 2025 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/clock_control/mspm0_clock_control.h>
+
+#include <ti/driverlib/driverlib.h>
+#include <string.h>
+
+#define MSPM0_ULPCLK_DIV COND_CODE_1(DT_NODE_HAS_PROP(DT_NODELABEL(ulpclk), clk_div),	\
```

Comment:
You changes the original PR code in some commits (pinctrl, gpio) by re-indenting (to fit 80 chars I guess?), but here with new macro you don't follow the same rule?